reactioncommerce/reaction
# Add Verification Url to Welcome Email

**Resolves**
#2125 

**Testing**
1. Clone from dev
1. Add email provider on dashboard
1. Sign as a new user
1. Follow the get started button on the welcome email
1. Observe that the user's document in the database is now showing `true` for the verified field
